lib/openstax/aws/stack.rb

```diff
@@ -338,6 +338,52 @@ def status
       end
     end
 
+    def status_reason
+      begin
+        aws_stack.stack_status_reason
+      rescue Aws::CloudFormation::Errors::ValidationError => ee
+        case ee.message
+        when /Stack.*does not exist/
+          "DOES_NOT_EXIST"
+        else
+          raise
+        end
+      end
```

Comment:
Its done like that in the function above, but what's the reason the error doesn't get forwarded?

Reply:
There are several reasons, `DOES_NOT_EXIST` is just one of the main ones... that's why there's an else with a `raise`, for this particular PR, this method just returns the `LAST` status reason of the stack events